[2025-01-31T03:18:39.619+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T03:18:39.644+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_files_branch_task manual__2025-01-31T03:18:23.061581+00:00 [queued]>
[2025-01-31T03:18:39.657+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_files_branch_task manual__2025-01-31T03:18:23.061581+00:00 [queued]>
[2025-01-31T03:18:39.658+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-01-31T03:18:39.676+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): check_files_branch_task> on 2025-01-31 03:18:23.061581+00:00
[2025-01-31T03:18:39.693+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=403) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T03:18:39.695+0000] {standard_task_runner.py:72} INFO - Started process 405 to run task
[2025-01-31T03:18:39.696+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_and_insert_csv_to_postgres_with_branch_adjusted', 'check_files_branch_task', 'manual__2025-01-31T03:18:23.061581+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/read_file_updated.py', '--cfg-path', '/tmp/tmpgh5n_u4i']
[2025-01-31T03:18:39.697+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask check_files_branch_task
[2025-01-31T03:18:39.747+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_files_branch_task manual__2025-01-31T03:18:23.061581+00:00 [running]> on host e2498a6c8083
[2025-01-31T03:18:39.820+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_and_insert_csv_to_postgres_with_branch_adjusted' AIRFLOW_CTX_TASK_ID='check_files_branch_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T03:18:23.061581+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T03:18:23.061581+00:00'
[2025-01-31T03:18:39.822+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T03:18:39.823+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T03:18:39.823+0000] {logging_mixin.py:190} INFO - Current task name:check_files_branch_task state:running start_date:2025-01-31 03:18:39.645010+00:00
[2025-01-31T03:18:39.824+0000] {logging_mixin.py:190} INFO - Dag name:validate_and_insert_csv_to_postgres_with_branch_adjusted and current dag run status:running
[2025-01-31T03:18:39.824+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T03:18:39.825+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-31T03:18:39.828+0000] {logging_mixin.py:190} INFO - Files found: ['invalid_records.csv', 'valid_records.csv']
[2025-01-31T03:18:39.841+0000] {python.py:240} INFO - Done. Returned value was: validate_records_task
[2025-01-31T03:18:39.842+0000] {branch.py:38} INFO - Branch into validate_records_task
[2025-01-31T03:18:39.843+0000] {skipmixin.py:230} INFO - Following branch ('validate_records_task',)
[2025-01-31T03:18:39.855+0000] {skipmixin.py:278} INFO - Skipping tasks [('no_files_task', -1)]
[2025-01-31T03:18:39.881+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T03:18:39.882+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=validate_and_insert_csv_to_postgres_with_branch_adjusted, task_id=check_files_branch_task, run_id=manual__2025-01-31T03:18:23.061581+00:00, execution_date=20250131T031823, start_date=20250131T031839, end_date=20250131T031839
[2025-01-31T03:18:39.891+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T03:18:39.892+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T03:18:39.893+0000] {logging_mixin.py:190} INFO - Dag name:validate_and_insert_csv_to_postgres_with_branch_adjusted queued_at:2025-01-31 03:18:23.079426+00:00
[2025-01-31T03:18:39.893+0000] {logging_mixin.py:190} INFO - Task hostname:e2498a6c8083 operator:BranchPythonOperator
[2025-01-31T03:18:39.911+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T03:18:39.945+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T03:18:39.949+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
